lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
@brettz9 ![](http://mazumafykc.com/wp-content/uploads/2014/11/homer-computer-doh.jpg)

Thanks SOO MUCH for the input!

Also a bigger fan of the `~` for index searches. Nobody gives love to the tilde. :-) What say you?
`if (~data.indexOf(']]>') { }`
https://www.joezimjs.com/javascript/great-mystery-of-the-tilde/